config/envVarCheck.ts

```diff
@@ -0,0 +1,26 @@
+import { env } from "../src/typeDefinitions/default.types";
+import { loadEnv } from "./config";
+
+/**
+ * Validate if all the required environment variables are set to a non empty value
+ * else throw an error
+ * ---
+ */
+export function validateEnv() {
+  //pass empty object as env and fromWorkerEnv = false, since this method is should get executed in github actions and not in worker
+  const env: env = loadEnv({}, false);
+  const missingEnvVariables = Object.keys(env).filter(
+    (key) => env[key] == ""
+  );
+
+  // Logging missing environment variables and throw error if any are missing
```

Comment:
You can remove these comments also?

Reply:
sure, these are redundant i think as the function level comments already convey the same info